ceph.spec.in

```diff
@@ -72,7 +72,7 @@ Epoch:		1
 %global _epoch_prefix %{?epoch:%{epoch}:}
 
 Summary:	User space components of the Ceph file system
-License:	LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and GPL-2.0-with-autoconf-exception and BSD-3-Clause and MIT
+License:	LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and GPL-2.0 WITH Autoconf-exception-2.0 and BSD-3-Clause and MIT
```

Comment:
Just a random thought: we no longer use autoconf, do we still need the exception bit?